release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1110,6 +1114,12 @@ link:https://issues.redhat.com/browse/OCPBUGS-39428[(*OCPBUGS-39428*)]
 [id="ocp-4-17-node-tuning-operator-bug-fixes_{context}"]
 ==== Node Tuning Operator (NTO)
 
+[discrete]
+[id="ocp-4-17-observability-bug-fixes_{context}"]
+==== Observeability
+
+* Previously,  when you log on under the *Administrator* perspective on the {product-title} web console and use the *Observe* -> *Alerting* function,  an "S is not a function" displayed on alert metrics graph. This issue happened because of a missing function validation check. With this release, the function validation check is added so the alert metric chart displays colelcted metrics.   (link:https://issues.redhat.com/browse/OCPBUGS-37291[*OCPBUGS-37291])
```

Comment:
```suggestion
* Previously,  when you log on under the *Administrator* perspective on the {product-title} web console and use the *Observe* -> *Alerting* function,  an "S is not a function" displayed on alert metrics graph. This issue happened because of a missing function validation check. With this release, the function validation check is added so the alert metric chart displays collected metrics.   (link:https://issues.redhat.com/browse/OCPBUGS-37291[*OCPBUGS-37291])
```

Reply:
:heavy_check_mark: